### PR TITLE
Docker Compose Environment pass-through

### PR DESCRIPTION
**Pull request by adamgering**
Pass through AES and Secret key from .env

Minor:
Removed verbose logging flags in docker compose.
Add some more binary files to .gitattributes
Fix line endings in docs/make.bat (also caused by .gitattributes)

### CODE REVIEW

**Comment by nadiabahrami** [comment]
One PR is one commit.  If it is different fixes they should be different PRs.  That is the standard we have been using

**Comment by adamgering** [comment]
The commit (and related commit message) on a file should represent why the file is change and not be unrelated. That way, when you view the change history for that individual file, it makes sense. The point of practices is to maintain a clean git commit history, both for the whole project and for individual files.

The standards we are trying to follow or those of the Linux Foundation. Another LF maintainer on this repo asked us explicitly not to squash unrelated commits together, and that multiple commits in a PR is fine.

PRs are separated if there are either two many files to review, they need to be reviewed separately, they are expected to be contentious, or it is likely they will need to be amended (as otherwise only the last commit is easily amended). None of that applies here.

File: docker-compose.yaml [hunk 1]
```diff
@@ -93,6 +93,8 @@ services:
     environment:
       - SERVER_HOST=${SERVER_HOST:-0.0.0.0}
       - SERVER_PORT=${SERVER_PORT:-8000}
+      - SECRET_KEY=${SECRET_KEY}
```
[the remark this answers is not in the document]
Comment (adamgering) [reply docker-compose.yaml]: The value configured in .env will not be passed into the docker container without putting it in the environment section of docker compose. This file change was missed in PR #259.

[the remark this answers is not in the document]
Comment (kthblmfld) [reply docker-compose.yaml]: Very well. In it goes.

**Pull request merged**